azkaban-web-server/src/main/java/azkaban/flowtrigger/DependencyInstanceContext.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.flowtrigger;
+
+/**
+ * This class holds the context info
+ * for a running dependency instance.
+ */
+public class DependencyInstanceContext {
```

Comment:
Why isn't it an interface?

Reply:
I still haven't thought about the implementation detail yet. Will create an interface once I see a need.